tests/test_logger.py

```diff
@@ -67,51 +72,61 @@ def test_set_level(self):
-        self.assertEqual(prev_level, app.handler.level)
-        self.assertEqual(prev_level, root_logger.level)
+        app.setLevel(app.frameHandler, non_level)
+        app.setLevel(app.fileHandler, non_level)
+        self.assertEqual(app.frameHandler.level, prev_level)
+        self.assertEqual(app.fileHandler.level, prev_level)
+        self.assertEqual(root_logger.level, prev_level)
 
     def test_frames(self):
-        app = logger.LoggerApp(name="name", parent=QObject())
+        app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
         self.assertEqual(app.frames(), (app.loggerFrame,))
 
+    def test_namer(self):
+        app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
+        name = "hell.logo"
+        modified_name = app.fileHandler.namer(name)
+        self.assertEqual(modified_name, "hello.log")
+
     def test_call_check_to_clear(self):
         with mock.patch("iquip.apps.logger.LoggerApp.checkToClear") as mocked_method:
-            app = logger.LoggerApp(name="name", parent=QObject())
+            app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
             app.loggerFrame.clearButton.clicked.emit()
             mocked_method.assert_called_once()
 
     def test_check_to_clear(self):
         with mock.patch("iquip.apps.logger.QWidget.show") as mocked_show:
-            app = logger.LoggerApp(name="name", parent=QObject())
+            app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
             app.checkToClear()
             mocked_show.assert_called_once()
 
     def test_call_clear_log(self):
         with mock.patch("iquip.apps.logger.LoggerApp.clearLog") as mocked_method:
-            app = logger.LoggerApp(name="name", parent=QObject())
+            app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
             app.confirmFrame.buttonBox.accepted.emit()
             mocked_method.assert_called_once()
 
     @mock.patch("iquip.apps.logger.QTextEdit.clear")
     def test_clear_log(self, mocked_clear):
         with mock.patch("iquip.apps.logger.logger") as mocked_logger:
-            app = logger.LoggerApp(name="name", parent=QObject())
+            app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
             app.clearLog()
             mocked_clear.assert_called_once()
             mocked_logger.info.called_once_with("Tried to clear logs by clicking clear button")
 
     def test_add_log(self):
+        app = logger.LoggerApp(name="name", logFilePath="", parent=QObject())
         with mock.patch("iquip.apps.logger.QTextEdit.insertPlainText") as mocked_insert:
             with mock.patch("iquip.apps.logger.QDateTime.currentDateTime") as mocked_time:
-                app = logger.LoggerApp(name="name", parent=QObject())
                 app.addLog("hello")
                 timeString = mocked_time().toString("yyyy-MM-dd HH:mm:ss")
                 msg = f"{timeString}: hello\n"
```

Comment:
Oh, now I realize it.
Please add an empty line between this class and `if __name__ == "__main__":`